api/src/identity-access-management/domain/services/refresh-token-service.js

```diff
@@ -35,8 +35,19 @@ async function findByUserId(userId) {
  * @param {function} params.uuidGenerator
  * @return {Promise<string>}
  */
-async function createRefreshTokenFromUserId({ userId, scope, source, uuidGenerator = randomUUID }) {
-  const expirationDelaySeconds = config.authentication.refreshTokenLifespanMs / 1000;
+async function createRefreshTokenFromUserId({
+  userId,
+  scope,
+  source,
+  uuidGenerator = randomUUID,
+  refreshTokenTemporaryStorage,
+  userRefreshTokensTemporaryStorage,
+}) {
+  refreshTokenTemporaryStorage = refreshTokenTemporaryStorage || globalRefreshTokenTemporaryStorage;
+  userRefreshTokensTemporaryStorage = userRefreshTokensTemporaryStorage || globalUserRefreshTokensTemporaryStorage;
```

Comment:
```suggestion
  refreshTokenTemporaryStorage ??= globalRefreshTokenTemporaryStorage;
  userRefreshTokensTemporaryStorage ??= globalUserRefreshTokensTemporaryStorage;
```

cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment

Reply:
On en avait discuter pendant la session de mob programming, et l'équipe trouvait ce type d'écriture (`??=`) moins parlante que l'utilisation directe de l'opérateur logique.